acestep/training/dataset_builder_modules/preprocess.py

```
@@ -21,6 +30,19 @@
 
+def _empty_gpu_cache() -> None:
+    """Release cached GPU memory (CUDA / MPS / XPU)."""
+    if torch.cuda.is_available():
+        torch.cuda.empty_cache()
+    if hasattr(torch, "xpu") and torch.xpu.is_available():
+        torch.xpu.empty_cache()
+    if hasattr(torch, "mps") and hasattr(torch.mps, "empty_cache"):
+        try:
+            torch.mps.empty_cache()
+        except Exception:
+            pass
```

Ruff (0.15.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)